firebase-firestore/src/main/java/com/google/firebase/firestore/util/AsyncQueue.java

```diff
@@ -68,6 +68,8 @@
      * set timeout, rather than waiting indefinitely for success or failure.
      */
     ONLINE_STATE_TIMEOUT,
+    /** A timer used to periodically attempt LRU Garbage collection */
+    GARBAGE_COLLECTION_DELAY
```

Comment:
Nit: Maybe remove `DELAY` since this represents the actual task, and not how long we are waiting for it.